common-lib/src/main/scala/com/gu/mediaservice/lib/aws/Kinesis.scala

```diff
@@ -51,7 +52,7 @@ class Kinesis(config: KinesisSenderConfig) extends GridLogging{
       logger.info(s"Published kinesis message: $result")
     } catch {
       case e: Exception =>
-        logger.error(s"kinesis putRecord exception message: ${e.getMessage}")
+        logger.error(markers, s"kinesis putRecord failed for ID: ${message.id} File: ${message.image.map(_.source.file)}", e)
```

Comment:
ID get's [pulled out as a marker](https://github.com/guardian/grid/blob/2c8c1ef40afc7fea3743fe764cac216ed79c0888/common-lib/src/main/scala/com/gu/mediaservice/lib/aws/ThrallMessageSender.scala#L57) too so having it in the message may be necessary? Makes searching easier, I guess.

I wonder if `UpdateMessage` should [expose](https://github.com/guardian/grid/blob/2c8c1ef40afc7fea3743fe764cac216ed79c0888/common-lib/src/main/scala/com/gu/mediaservice/lib/aws/ThrallMessageSender.scala#L53-L61) more information as markers?

Reply:
I've moved the fileName to the message marker now whilst retaining the exception. Does that look better @akash1810 ?